packages/welldone-wallet/src/lib/welldone.ts

```diff
@@ -0,0 +1,328 @@
+import { transactions as Transactions, utils } from "near-api-js";
+import {
+  WalletModuleFactory,
+  InjectedWallet,
+  WalletBehaviourFactory,
+  Action,
+  FinalExecutionOutcome,
+} from "@near-wallet-selector/core";
+import {
+  SignAndSendTransactionParams,
+  ViewAccessKeyParams,
+  WelldoneWalletParams,
+  WelldoneWalletState,
+} from "./injected-welldone";
+import icon from "./icon";
+import { createAction } from "@near-wallet-selector/wallet-utils";
+
+async function initWalletState(): Promise<WelldoneWalletState> {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const wallet = (window as any).dapp;
```

Comment:
Instead of using `(window as any).dapp` can you declare something like this at the top of the file: 

```ts
declare global {
  interface Window {
    dapp: WalletProvider;
  }
}
```

Reply:
ok I'll fix it!